pisa/stages/osc/prob3numba/numba_osc.py

```diff
@@ -89,16 +89,21 @@ def test_get_H_vac():
 
 
 @myjit
-def get_H_mat(rho, nsi_eps, nubar, H_mat):
+def get_H_mat(rho, mat_pot, nubar, H_mat):
     ''' Calculate matter Hamiltonian in flavor basis
 
     Parameters:
     -----------
     rho : float
-        density
+        electron number density (in moles/cm^3)
+        (numerically, this is just the product of
+        electron fraction and mass density in g/cm^3,
+        since the number of grams per cm^3 corresponds to
+        the number of moles of nucleons per cm^3)
 
-    nsi_eps : complex 2-d array
-        Non-standard interaction terms
+    mat_pot : complex 2-d array
```

Comment:
please include in docstring that one should make this `np.diag([1, 0, 0])` for standard interactions only (here and anywhere else this kwarg is used). Also, make the docstrings consistent for this wherever they appear.